### DISCUSSION TITLE
[Issue] Copilot isnt getting results due to a self signed cert

### DISCUSSION THREAD

**Discussion by s0r3-glitch** [community/community#6723] [Copilot Conversations] [answered]
So I had an issue getting copilot to work see in discussion [6707](https://github.com/github/feedback/discussions/6707) and was finally able to get it to work by going back to a older version and I am currently running v1.0.1896 and I am fairly confident its going to be in the current build since the release notes dont mention anything about it. When copilot trys and get results it erroring out with it saying `Error on ghost text request Error: self signed certificate in certificate chain` as a result I am not getting any solutions from the AI. What can I do with copilot to get it to ignore the self signing cert error and just get the results?

**Comment by davecheney** [comment, marked as answer]
Please don’t use older versions of copilot, they won’t work as the protocols and servers have changed. https://github.com/github/feedback/discussions/6637

**Reply by s0r3-glitch** [reply]
ok thank you
